### ISSUE TITLE
Fix template's load_data with input_dir when called from installed package

### ISSUE THREAD

**Issue by micahjsmith** [HDI-Project/ballet#61]
* ballet version: 0.7.9

### Description

If a ballet project is rendered from the default project template, and then installed using pip, the user is not working from the source directory. Thus the project cannot rely on `from ballet.project import config`

### What I Did

Basically:

```
pip install -e ballet-predict-census-income
cd /tmp/work
python -c "from predict_census_income.api import api; api.load_data('/path/to/input/data/dir')"
Traceback (most recent call last):
  File "main.py", line 60, in <module>
    main()
  File "main.py", line 55, in main
    create_automl_training_data(b, datadir)
  File "main.py", line 31, in create_automl_training_data
    X_df, y_df = b.api.load_data(path / 'train')
  File "/work/.venv/lib/python3.8/site-packages/ballet/project.py", line 327, in load_data
    return self._load_data(*args, **kwargs)
  File "/work/.venv/lib/python3.8/site-packages/funcy/calc.py", line 121, in wrapper
    result = func(*args, **kwargs)
  File "/work/.venv/lib/python3.8/site-packages/predict_census_income/load_data.py", line 13, in load_data
    entities_config = some(where(tables, name=entities_table_name))
  File "/work/.venv/lib/python3.8/site-packages/funcy/colls.py", line 322, in where
    return filter(match, mappings)
TypeError: 'NoneType' object is not iterable
```

**Comment by micahjsmith** [comment]
Look into https://pypi.org/project/lazy-object-proxy/ with `config = Proxy(partial(load_config_in_dir, getcwd()))`

**Comment by micahjsmith** [comment]
Should be fixed in de5cc37, still testing

**Comment by micahjsmith** [comment]
Confirmed resolved
